pages/archiving/setup.rst

```diff
@@ -56,6 +56,127 @@ The archived indices will be stored in a backend. A backend that stores the data
+Graylog writes these segments, they are uploaded. 
+
+Keep in mind you need 5 gigabytes of free disk space in this directory to manage 10 segments at 
+500 megabytes. However, it may use less memory because it deletes files as soon as they upload.
+
+AWS Region
+~~~~~~~~~~
+
+By default, Graylog pulls the region from your file systems. 
+
+If you choose to select a region, you should use the one where you created the bucket. 
+
+If you are not using AWS, you do not need to configure this. 
+
+S3 Output Base Path
+~~~~~~~~~~~~~~~~~~~
+
+This is a prefix to the file name that works similar to a directory. Configuring this will 
+help you organize data. 
+
+You can use the following variable to construct a dynamic value for each archive to give 
+it structure:
+
++-----------------------------+-------------------------------------------+
+| variable                    | Description                               | 
++=============================+===========================================+
+| index-name                  | Name of the index that gets archived      |     
++-----------------------------+-------------------------------------------+
+| year                        | Archival date year                        |     
++-----------------------------+-------------------------------------------+                                                
+| month                       | Archival date month                       |                                                
++-----------------------------+-------------------------------------------+
+| day                         | Archival date day                         |     
++-----------------------------+-------------------------------------------+                                                     
+| hour                        | Archival date hour                        |     
++-----------------------------+-------------------------------------------+     
+| minute                      | Archival date minute                      |     
++-----------------------------+-------------------------------------------+     
+| second                      | Archival date second                      |
++-----------------------------+-------------------------------------------+
+
+Activate Backend
+~~~~~~~~~~~~~~~~
+
+After configuring your bucket, click *Save*.
+
+This will bring you back to the *Edit archive backend configuration* page. 
+
+To activate the backend, you need to:
+
+#. Click on the *Configuration* tab located in the top righthand corner.
+#. Under the *Backend* dropdown menu, select the backend you want to activate. 
+#. You can choose to change configurations or use the defaults provided. 
+#. Click the green *Update configuration* button at the bottom of the screen.
+#. This will return you to the *Archives* screen.
+
+The archiving will run automatically.  
+
+To review your S3 archive, click the *Archive Index* button. 
```

Comment:
```suggestion
```

I would not mention that here.
This isn't specific to the backend, and is probably covered in the Usage section